maths/max_sectors_of_circle.py

```diff
@@ -0,0 +1,23 @@
+def max_sectors_of_circle(num_cuts: float) -> float:
+    """
+    returns the maximum amount of
+    sectors a circle can be divided
+    by if cut 'num_cuts' times
+
+    >>> max_sectors_of_circle(54)
+    1486.0
+    >>> max_sectors_of_circle(7)
+    29.0
+    >>> max_sectors_of_circle(22.5)
+    265.375
+    >>> max_sectors_of_circle(-222)
+    -1
+    """
+
```

Comment:
Consider adding a check for the input type of `num_cuts` to ensure it is a numeric value. You can use the `isinstance` function to check if `num_cuts` is a float or an integer. This will help prevent potential type errors when the function is used.

Reply:
Thank you for your suggestion! Already implemented and added tests for this :)